### ISSUE TITLE
Fix the logo accessibility

### ISSUE THREAD

**Issue by emmabloom04** [emmabloom04/kizuna-rail#1]
https://github.com/emmabloom04/kizuna-rail/blob/23b59a07771e39338c881e18175e1256cd5ce145/src/views/partials/header.ejs#L28

The site's logo in the header is missing a proper title attribute for hover tooltips, which helps with accessibility and user experience.
